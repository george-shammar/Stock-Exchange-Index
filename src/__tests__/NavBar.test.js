// import React from 'react';
// import NavBar from '../components/Navbar';

import React from 'react';
import ReactDom from 'react-dom';
import NavBar from '../components/Navbar';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

it('renders NavBar without crashing', () => {
  const div = document.createElement('div');

  ReactDom.render(
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>, 
  div);

  ReactDom.unmountComponentAtNode(div);
});

it('renders the AllStocks component correctly', () => {
  const tree = renderer.create(<BrowserRouter>
    <NavBar />
  </BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});