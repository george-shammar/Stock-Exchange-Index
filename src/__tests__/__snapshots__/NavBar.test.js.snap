// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the AllStocks component correctly 1`] = `
<div
  className="d-flex mb-3 border"
>
  <h1
    className="app-name"
  >
    Stochastics
  </h1>
  <a
    className="mx-5 btn btn-outline-primary"
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
</div>
`;
